docs/virtual_machines/disks_and_volumes.md

```diff
@@ -412,6 +412,62 @@ spec:
         claimName: mypvc
 ```
 
+
+#### Statically provisioned block PVCs
+
+To use an externally managed local block device from a host ( e.g. /dev/sdb , zvol, LVM, etc... ) in a VM directly,
+you would need a provisioner that supports block devices, such as OpenEBS LocalPV.
+
+Alternatively, local volumes can be provisioned by hand.
+I.e. the following PVC:
+
+```yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: myblock
+spec:
+  storageClassName: local-device
+  volumeMode: Block
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 100Gi
+```
+
+can claim a PersistentVolume pre-created by a cluster admin like so:
+
+```yaml
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
```

Comment:
I haven't tried this in awhile but am surprised that the `StorageClass` resource is necessary.  What happens if it is omitted?

Reply:
it'll say it doesnt know the storageclass